coretex/cli/modules/node.py

```diff
@@ -200,6 +200,35 @@ def shouldUpdate(image: str) -> bool:
     return True
 
 
+def getNodeVersion(image: Optional[str] = None) -> str:
```

Comment:
Image should be removed as param from here. It should be read from the running container (if any) and taken from config as a failback.